sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py

```diff
@@ -737,11 +742,12 @@ def connect(self):
             ) from None
         try:
             self.sock = create_connection(
-                url="wss://{}".format(self._custom_endpoint or self._host),
+                url="wss://{}".format(self._custom_endpoint or self._host) if self._use_tls
+                    else "ws://{}".format(self._custom_endpoint or self._host),
```

Comment:
no changes need to be made in the _pyamqp_transport file to support this? or expose the option to not use tls in public api?

Reply:
I don't believe we have an emulator for SB at this time so use_tls wouldn't be used, I don't believe it is a part of public api regardless -- it is taken from the conn_str parsing in EH

Reply:
sounds good, we'll prob end up adding stuff when we need to for features anyway